internal/beater/otlp/map_caching.go

```diff
@@ -0,0 +1,71 @@
+package otlp
```

Comment:
Should this file go into r8 package instead? Since this is more about r8 instead of about otlp.

Reply:
Makes sense, I've moved it to R8.